guides/common/modules/proc_enabling-pull-based-transport.adoc

```diff
@@ -0,0 +1,40 @@
+[id="enabling-pull-based-tranport_{context}"]
+= Enabling Pull-based Transport
+
+.Procedure
+If your {Project} or {SmartProxy} is preinstalled, enable the pull provider using:
+ifdef::satellite[]
+`satellite-installer --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt`
+endif::[]
+`foreman-installer --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt`.
```

Comment:
This should probably be hidden by an ifdef/ifndef so we won't get two commands in satellite secnario.

Reply:
@adamruzicka Is there a specific link I should put in?  I just pulled the github link from the demo.

Reply:
I assume this question was supposed to be attached to this comment https://github.com/theforeman/foreman-documentation/pull/1584#discussion_r957163138 . 

I'd be inclined to say no. At this point, people should know what client repos are and where to get them? In [here](https://github.com/theforeman/foreman-documentation/pull/1584/files#diff-12c27ae804ff7c1fc49f65f0010f9c5a7e491f5785c16dae7548fad446432279R24) you just refer to them without explicitly stating where they live.